packages/teams-js/src/public/dialog.ts

```diff
@@ -79,7 +79,7 @@ export namespace dialog {
     ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task, FrameContexts.meetingStage);
 
     // Send tasks.completeTask instead of tasks.submitTask message for backward compatibility with Mobile clients
-    sendMessageToParent('tasks.completeTask', [result, Array.isArray(appIds) ? appIds : [appIds]]);
+    sendMessageToParent('tasks.completeTask', [result, appIds ? (Array.isArray(appIds) ? appIds : [appIds]) : []]);
```

Comment:
I cannot think of any scenarios, that can break v1 compatibility with this change. If you do, please let me know! :)

Reply:
This shouldn't break back compat, but I would check to see what is expected from the hosts perspective. Maybe they are expecting an `undefined` in an array? Maybe it's better to just send `undefined`?

Reply:
Should we add a UT around this?

Reply:
Good idea! Added a unit test for this!